### PR TITLE
[dagster-tableau] Add basic docs

### PR DESCRIPTION
**Pull request by maximearmstrong**
## Summary & Motivation

Introduce initial docs for Tableau, to send to initial design partners. These are hidden from nav for now since the integration is not being widely publicized.

## Changelog

NOCHANGELOG

- [ ] `NEW` _(added new feature or capability)_
- [ ] `BUGFIX` _(fixed a bug)_
- [ ] `DOCS` _(added or updated documentation)_

### CODE REVIEW

**Comment by maximearmstrong** [comment]
* **#24609** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/24609?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `master`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @maximearmstrong and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: docs/content/integrations/tableau.mdx [hunk 1]
```diff
@@ -0,0 +1,128 @@
+---
+title: "Using Tableau"
```
Comment (benpankow) [inline docs/content/integrations/tableau.mdx:2]:
```suggestion
title: "Using Dagster with Tableau"
description: Represent your Tableau assets in Dagster
```

Update title based on Sandy's feedback in https://github.com/dagster-io/dagster/pull/24269

File: docs/content/integrations/tableau.mdx [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+---
+
+# Using Tableau
```
Comment (benpankow) [inline docs/content/integrations/tableau.mdx:5]:
```suggestion
# Using Dagster with Tableau
```

File: docs/content/integrations/tableau.mdx [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+
+- A Tableau site, either on Tableau Cloud or Tableau Server
+
+- A connected app configured to access Tableau. For more information, see [Use Tableau Connected Apps for Application Integration](https://help.tableau.com/current/online/en-us/connected_apps.htm) in the Tableau documentation.
+
+</details>
+
+## Represent Tableau assets in the asset graph
+
+To load Tableau assets into the Dagster asset graph, you must first construct a Tableau resource, which allows Dagster to communicate with your Tableau workspace. The Tableau resource to create depends on your Tableau deployment type - `TableauCloudWorkspace` for Tableau Cloud and `TableauServerWorkspace` for Tableau Server. To connect to the Tableau workspace, you'll need to [configure a connected app with direct trust](https://help.tableau.com/current/online/en-gb/connected_apps_direct.htm) in Tableau, then supply your Tableau site information and connected app credentials to the resource. The Tableau resource uses the JSON Web Token (JWT) authentication to connect to the Tableau workspace.
```
Comment (benpankow) [inline docs/content/integrations/tableau.mdx:32]:
```suggestion
To load Tableau assets into the Dagster asset graph, you must first construct a Tableau resource, which allows Dagster to communicate with your Tableau workspace. The Tableau resource to create depends on your Tableau deployment type - use `TableauCloudWorkspace` if you are using Tableau Cloud or `TableauServerWorkspace` if you are using Tableau Server. To connect to the Tableau workspace, you'll need to [configure a connected app with direct trust](https://help.tableau.com/current/online/en-gb/connected_apps_direct.htm) in Tableau, then supply your Tableau site information and connected app credentials to the resource. The Tableau resource uses the JSON Web Token (JWT) authentication to connect to the Tableau workspace.
```

nit: suggestion to make it more clear that you use one or the other

File: docs/content/integrations/tableau.mdx [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+
+Dagster can automatically load all data sources, sheets, and dashboards from your Tableau workspace. Call the `build_defs()` function, which returns a `Definitions` object containing all the asset definitions for these Tableau assets.
+
+```python file=/integrations/tableau/representing-tableau-assets.py
+from dagster_tableau import TableauCloudWorkspace, TableauServerWorkspace
+
+from dagster import EnvVar
+
+# Connect to Tableau Cloud using the connected app credentials
```
Comment (benpankow) [inline docs/content/integrations/tableau.mdx:41]: Not blocking merging, but could be nice to toggle between the two using a tab group, e.g. https://github.com/dagster-io/dagster/blob/master/docs/content/concepts/assets/external-assets.mdx#L44

**Review by benpankow — APPROVED** [review]

**Pull request merged**